net-misc/nyx/nyx-2.1.0-r1.ebuild

```diff
@@ -24,5 +25,5 @@ RESTRICT="!test? ( test )"
 DEPEND="
 	dev-python/setuptools[${PYTHON_USEDEP}]"
 RDEPEND="
-	net-libs/stem
+	net-libs/stem[${PYTHON_USEDEP}]
```

Comment:
You didn't add `python3_8` support in https://github.com/gentoo/gentoo/pull/14745 so this can't work.

Do you want more time to fix that PR?

Reply:
Fixed now.